packages/scoped-elements/package.json

```diff
@@ -14,6 +14,7 @@
   "author": "open-wc",
   "homepage": "https://github.com/open-wc/open-wc/tree/master/packages/scoped-elements",
   "main": "index.js",
+  "type": "module",
```

Comment:
Pretty sure this will count as a breaking change.

Reply:
I don't think so. Bundlers look at the `"exports"` without taking the `"type"` into account.
NodeJS does take the `"type"` into account, but fails as it tries to interpret the file as a CommonJS module.

So as far as I can see (and please correct me if I'm wrong) it's a bugfix, as it doesn't restrict anything, and fixes something that currently doesn't work when importing it from NodeJS.

Reply:
Im not sure if this is a breaking change, I think adding type: module is more "dangerous" for tooling packages/projects than packages like this, which is runtime/client only. As Jesse also mentioned, I don't think adding this changes anything in the way the module is resolved. I _think_ we can ship this as a minor. @Westbrook do you agree?